anonymize.py

```diff
@@ -366,10 +366,10 @@ def delete_dates(cur_files):
                         help="Prefix for output files.")
     parser.add_argument("--output-dir", type=str, default = Path.cwd(),
                         help="Directory for output files")
-    parser.add_argument("--keep-dates", action="store_true",
-                        help="Keep the comment dates")
+    parser.add_argument("--remove-dates", action="store_true",
+                        help="Remove also the dates form tracked edits")
     parser.add_argument("--keep-initials", action="store_true",
-                        help="Keep the commenter initials")
+                        help="Keep the author's initials")
```

Comment:
I believe this refers not to the document author, but the comment authors' initials, so it might be better to specify that
```suggestion
                        help="Keep the comment authors' initials")
```